qualtran/bloqs/arithmetic.py

```diff
@@ -1142,4 +1142,4 @@ def signature(self) -> Signature:
     def build_call_graph(self, ssa: 'SympySymbolAllocator') -> Set['BloqCountT']:
         # Take the sign qubit as a control and cnot the remaining qubits, then
         # add it to the remaining n-1 bits.
-        return {(TGate(), 4 * (self.bitsize - 2))}
+        return {(TGate(), (self.bitsize - 2))}
```

Comment:
This was the suspense @fdmalone 
```suggestion
        return {(Toffoli(), (self.bitsize - 2))}
```

Reply:
phew, thanks! A test would have checked me :)